Features/Context/PlatformUI.php

```diff
@@ -0,0 +1,327 @@
+    }
+
+    /**
+     * @Then I should see the following elements:
+     */
+    public function iSeeElementsVisible( TableNode $elements )
+    {
+        foreach ( $elements as $element )
+        {
+            $name = array_values( $element )[0];
+            $isVisible = $this->isElementVisible( $name, '.ez-zone-name, .ez-navigation-item' );
+            Assertion::assertTrue( $isVisible, "Element $name not found" );
+        }
+    }
+
+    /**
+     * @Then I shouldn't see the following elements:
+     * @Then I should not see the following elements:
+     */
+    public function iSeeElementsNotVisible( TableNode $elements )
+    {
+        foreach ( $elements as $element )
+        {
+            $name = array_values( $element )[0];
+            $isVisible = $this->isElementVisible( $name, '.ez-zone-name, .ez-navigation-item' );
+            Assertion::assertFalse( $isVisible, "Element $name found" );
+        }
+    }
+
+    /**
+     * @Then I should see (an) element :element with value :value
+     */
+    public function iSeeElementsWithValue( $element, $value )
+    {
+        $valueActual = $this->checksElementSiblingValue( $element, '.ez-fieldview-label' );
+        //check later
+        Assertion::assertEquals( preg_replace( '/\s+/', '', $value ), preg_replace( '/\s+/', '', $valueActual ) );
+    }
+
+    /**
+     * @Then I should see (an) element :element with (an) image
+     */
+    public function iSeeElementImage( $element )
+    {
+        //Temporary solution ONLY
+        sleep( 2 );
+        $imgUrl = $this->checksImagePresent( $element, '.ez-fieldview-label' );
+        $isImage = exif_imagetype( $imgUrl );
+        Assertion::assertNotFalse( $isImage, "Not an image" );
+    }
+
+    /**
+     * @Then I should see (an) element :element with (an) file :file
+     */
+    public function iSeeElementFile( $element, $file )
+    {
+        $url = $this->getFileUrl( $element, '.ez-fieldview-label' );
+        $fileContentActual = file_get_contents( $url );
+        $file = rtrim( realpath( $this->getMinkParameter( 'files_path' ) ), DIRECTORY_SEPARATOR ).DIRECTORY_SEPARATOR.$file;
+        $fileContentExpected = file_get_contents( $file );
+        Assertion::assertEquals( $fileContentActual, $fileContentExpected );
+    }
+
+    /**
+     * @Then I should see elements with the following names:
+     */
+    public function iSeeElements( TableNode $elements )
+    {
+        foreach ( $elements as $element )
+        {
+            $found = false;
+            $name = array_values( $element )[0];
+            $found = $this->checksElementsByText( $name, '.ez-selection-filter-item' );
+            Assertion::assertNotNull( $found, "Element: $name not found" );
+        }
+    }
+
+    /**
+     * @Then I should be logged in
+     */
+    public function iShouldBeLoggedIn()
+    {
+        $this->shouldBeLoggedIn = true;
+
+        $jsCode = <<<JSCODE
```

Comment:
eh what is that ? if it works, you are lucky, since the use callback function can be called asynchronously.
if you just want to check if an element is in the page, you don't need YUI at all, something like:

```js
return (document.querySelector('.ez-loginform') === null);
```

should do the trick. But that does not really check that you are logged but more that the login form disappeared.